recipes/bison/all/conanfile.py

```diff
@@ -4,7 +4,6 @@
 
 
 SIMPLE_OUTPUT = """\
-#!/usr/bin/env python
```

Comment:
Shouldn't the bincrafters-conventions tool only remove the shebang if it's the first line of a source?
Because the shebang is needed here. As is noted by the later commit that adds python.

Reply:
Fixed. Thanks for the ping @ericLemanissier 👍